### ISSUE TITLE
ICU in estonia

### ISSUE THREAD

**Issue by Q-lds** [koroonakaart/koroonakaart#32]
It would be interesting and useful to know:
- how many ICU beds are available in Estonia (and perhaps track their availability)
- number of available ICU units fitted with ventilators 

I immagine these information (at least the raw numbers) should be available somewhere.

**Comment by zidik** [comment]
Yes! It is much more informative and actionable that just the number of cases.

The **number of cases** does not give any actionable information. Furthermore, the number itself depends on the availability of the testing and who is being chosen for testing.
On the other hand, the **percentage of occupied ICU beds** in the country on tells directly how serious the situation is. As it approaches 80%, 90%, 100% the system breaks.

By looking at plot of occupied ICU beds, we can clearly predict, when will the healthcare system collapse under the pressure. -- **This**, is actionable.

**Comment by keeganmcbride** [comment]
We don't have this data now, but if it becomes available we will add it. I hope that @infokujur can help to get this information.

**Comment by Q-lds** [comment]
But the overall number of ICU beds should be available (or at least knowable), and we already have the number of people with corona in critical conditions (which usually also means ICU beds) as it is usually given in the daily updates by the health board.

Knowing how many ICU spaces are available at any given time may not be doable, but knowing the max capacity would have a pretty good idea of how many ICUs are (theoretically) available to non covid patients, and thus to track the covid induced saturation of ICUs beds.

**Comment by rivol** [comment]
the new science report has the numbers: https://www.valitsus.ee/et/uudised/covid-19-leviku-tokestamise-teadusnoukoja-raport

there doesn't seem to be a copy in English, but tl;dr is: 157 beds overall today, 80 of them available. 50 more could be added (by converting the lower-level ones), so total of 130 can be made available for Covid patients.

**Comment by keeganmcbride** [comment]
Thanks! Yes I read this today, the numbers are there I just have no idea how to visualize them. We wont get updated data on this I assume so everything would just be speculation from our part. If someone has a good idea on how to visualize then I can look into adding it.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by rivol** [comment]
hmmm... maybe add a second row of number to the statsbar thing? might also be useful to quickly show the daily increase (absolute or percentage) of other numbers.
here's a quick mockup:
![image](https://user-images.githubusercontent.com/1205451/77481755-fbc79980-6e2c-11ea-8d44-bb964f92fa39.png)

**Comment by timoteosk** [comment]
For visualisation purposes a horizontal (red) line could be added to cases graph at 130 cases (conservative estimate of available ICU places according to the cited report) or two lines at 130 cases and 500 cases (maximum according to the Health Board if all possible conversions done (https://epl.delfi.ee/uudised/koroonakriisi-koige-mustemale-stsenaariumile-tervishoiususteem-vastu-ei-pea?id=89407295)).
